addons/mail/static/src/new/core/messaging_service.js

```diff
@@ -536,15 +537,14 @@ export class Messaging {
                         this.store.threads[createLocalId("mail.channel", notif.payload.channel.id)];
                     const member = this.thread.insertChannelMember({
                         id: notif.payload.id,
-                        persona: this.partner.insertPersona({
-                            partner: this.partner.insert({
-                                id: notif.payload.persona.partner.id,
-                                name: notif.payload.persona.partner.name,
-                            }),
+                        persona: this.persona.insert({
+                            id: notif.payload.persona.partner.id,
+                            name: notif.payload.persona.partner.name,
+                            type: "partner",
```

Comment:
for later I guess but this should not "ignore" the (potential) "guest" payload from persona